.travis.yml

```diff
@@ -24,4 +24,4 @@ install:
     sudo ln -s $HOME/sbt/bin/sbt /usr/local/bin/sbt
 
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION ";scalafmtCheckAll ;test"
+  - sbt ++$TRAVIS_SCALA_VERSION ";scalafmtSbt; scalafmtAll; test"
```

Comment:
Shouldn't this be `scalafmtSbtCheck; scalafmtCheckAll` ?

Reply:
Of course! What was I thinking :smile_cat: